PocketKit/Sources/PocketKit/Home/HomeViewModel.swift

```diff
@@ -731,8 +733,10 @@ extension HomeViewModel {
         let viewModel = HomeItemCellViewModel(
             item: sharedWithYouItem.item,
             overflowActions: [ .share { [weak self] sender in
-                    self?.share(sharedWithYouItem, at: indexPath, with: sender)
-                }],
+                Task {
+                    await self?.share(sharedWithYouItem, at: indexPath, with: sender)
```

Comment:
Does this work since its ui and needs main thread?

Reply:
it does because `HomeViewModel` is marked as `@MainActor`

<p align=center>
<img width="800" src="https://github.com/Pocket/pocket-ios/assets/34376330/30474168-6180-455f-9d1f-836e3e26c12b">

</p>